plugins/content_types/search_result_count.inc

```diff
@@ -21,7 +21,16 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
   $block = new stdClass();
 
   $search_result = drupal_static('ting_search_results');
-  if (isset($search_result)) {
+
+  if(isset($search_result->error)) {
+    $txt = t('An error occurred: %error_text ', array('%error_text' => $search_result->error));
```

Comment:
I do not think relaying the error message to the end users is a particulary good idea.

1. They will probably not understand it
2. It is probably not something they can do anything about
3. It will not be translated to the site language.

Consequently I think we should leave it out.